pkgs/development/tools/nc4nix/default.nix

```
@@ -8,37 +8,21 @@
 
 buildGoModule {
   pname = "nc4nix";
-  version = "unstable-2023-11-06";
+  version = "unstable-2024-03-01";
```

```suggestion
  version = "0-unstable-2024-03-01";
```

version numbers must start with a digit

---

Oh thank you, wasn't aware of this